tutor/src/models/task-plans/teacher/tasking.js

```diff
@@ -103,14 +100,26 @@ class TaskingPlan extends BaseModel {
     return moment(this.due_at).isAfter(Time.now);
   }
 
+  @computed get isDueAfterOpen() {
+    return moment(this.due_at).isAfter(moment(this.opens_at));
+  }
+
+  @computed get isCloseAfterDue() {
+    return moment(this.closes_at).isAfter(moment(this.due_at));
+  }
+
```

Comment:
you don't need to make sure the argument to `isAfter` is a momentjs instance, it'll take care of that for you.  Should be able to: `moment(this.closes_at).isAfter(this.due_at)`

Reply:
good to know! :) thanks.